include/osw_config_keys.h

```diff
@@ -58,6 +58,7 @@ extern OswConfigKeyShort resetDay;
 #if OSW_PLATFORM_ENVIRONMENT_ACCELEROMETER == 1
 extern OswConfigKeyInt stepsPerDay;
 extern OswConfigKeyBool stepsHistoryClear;
+extern OswConfigKeyBool settingDisplay;
```

Comment:
Oh no! I've asked you to _prefix_ the chosen variable name, NOT to replace it entirely... Do you understand now? :rofl:

Reply:
@simonmicro hmm...`settingDisplayStepsGoal`?